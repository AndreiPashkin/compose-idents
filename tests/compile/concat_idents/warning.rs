//! Using `compose_idents!` should trigger a deprecation warning.
#![deny(warnings)]
use compose_idents::compose_idents;

compose_idents!(name = foo, {
    fn name() {}
},);

fn main() {}
