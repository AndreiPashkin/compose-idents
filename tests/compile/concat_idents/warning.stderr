error: use of deprecated macro `compose_idents`: Renamed to compose!. Use compose!(...) instead.
 --> tests/compile/concat_idents/warning.rs:5:1
  |
5 | compose_idents!(name = foo, {
  | ^^^^^^^^^^^^^^
  |
note: the lint level is defined here
 --> tests/compile/concat_idents/warning.rs:2:9
  |
2 | #![deny(warnings)]
  |         ^^^^^^^^
  = note: `#[deny(deprecated)]` implied by `#[deny(warnings)]`
